### ISSUE TITLE
Character face warping repeatedly while not touching the mouse or keyboard

### ISSUE THREAD

**Issue by Rekipstar** [imchillin/Anamnesis#1334]
**Describe the bug**
My character's face, ears, and hair suddenly started warping and changing, happening every few seconds to a minute, even if I was not touching my keyboard or mouse. (Have included a video of it happening, since it's a little hard to describe. Note at the end of the video her face returns to normal; this only lasted for a few seconds after ending the video, and only happened once in several minutes and at least 20 iterations of the warping.)

**To Reproduce**
Steps to reproduce the behaviour:
1. Load appearance data of my character, Deya (included in the .zip) onto player character
2. Load Yugiri NPC data onto Carbuncle
3. Enter gpose; I don't know if this matters, but every time I started by loading the Highlander Hyur Masculine default sit
4. Enable posing (the Rotation and Scale boxes were checked when this started, the Positioning box was not; unchecking Scale did not stop the behavior)
5. Rotate the neck and/or head (don't remember if it happened only after rotating both or not, but I was doing both)
6. After several seconds to a couple minutes, the head will start warping seemingly at random, even while not touching the keyboard or mouse or even having FFXIV in focus (it happened even while I was focused on Discord on my second monitor.)

**Repeatability**
How often does the bug occur:
Occurred with my plugins loaded (not until a while into the posing session)
Tried restarting the computer and launching without any of my Dalamud plugins, still happened (almost immediately after rotating the head)
Tried relaunching with plugins and using Ktisis rather than Anamnesis; there was no warping, which means this is an Anamnesis exclusive problem

This is very similar to another issue I reported... probably over a year ago now? That issue was fixed, but I'm noticing similar error messages in the log files.

**Expected behaviour**
No part of her should move or change without my input.

**Log file**
[bug report files.zip](https://github.com/imchillin/Anamnesis/files/12900687/bug.report.files.zip)
[2023-10-13-18-06-59.txt](https://github.com/imchillin/Anamnesis/files/12900695/2023-10-13-18-06-59.txt)
[2023-10-13-18-40-16.txt](https://github.com/imchillin/Anamnesis/files/12900696/2023-10-13-18-40-16.txt)

Drag and drop the related log file(s) into the window, and they will be uploaded as part of your bug report. Please avoid using screenshots, as they very rarely provide any information of use to the developers. If your issue would benefit from additional evidence, such as a pose or appearance file, please package them into a `.zip` archive and upload them as well.
